core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -109,9 +110,23 @@ def __init__(
                 role=role,
                 memory_size=memory_size,
                 timeout=timeout,
+                environment={
```

Comment:
Are you going to update the handler as a part of this?

Reply:
In the workshop you mean? Yes, once we release

Reply:
Ah, I forgot we no longer provide dummy handlers